src/App.tsx

```diff
@@ -43,6 +46,20 @@ function getWsUri(id: string) {
   );
 }
 
+function useKeyboardCtrlIntercept(key: string, reaction: (evt: KeyboardEvent) => unknown) {
+  useEffect(() => {
+    const wrappedReaction: typeof reaction = (evt) => {
+      if (!(evt.ctrlKey && evt.key.toLowerCase() === key.toLowerCase())) return;
+      evt.preventDefault();
+      reaction(evt);
+    }
+    const controller = new AbortController();
```

Comment:
What is the AbortController used here for? Is there any advantage over just using `removeEventListener()`?

Reply:
on an earlier version I used removeEventListener and it didn't work properly.

It turned out that I had another issue (I didn't return the cleanup closure, wrote a bit too much Rust of late :-P), but before I figured that out I tried this.

After I fixed that issue I kept it, because it seems more expressive and less error-prone to me. removeEventListener relies on reference stability for the callback, and you need to explicitly mention the cleanup when binding the Callback.

Here it is on [MND](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#add_an_abortable_listener).